docs/docsource/Reference/bucket_ingestion_metrics/get_throughput_count_per_location.rst

```diff
@@ -0,0 +1,22 @@
+GET Count per Location
```

Comment:
Calling this file `throughput_count` might be a little confusing since we also have the `throughput_rate` file - maybe we could rename the file to specify `get_completions_per_location.rst`

Reply:
Also this file looks like a duplicate of `get_ingested_object_count`

Reply:
done, and fixed.